lib/api/kuzzle.js

```diff
@@ -232,6 +238,33 @@ class Kuzzle extends EventEmitter {
 
         return updated;
       });
+=======
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Parse error at line 241 column 1:

231: 
232:     return this.pluginsManager.pipe(events, data)
233:       .then(updated => {
234:         let i; // NOSONAR
235:         for (i = 0; i < events.length; i++) {
236:           super.emit(events[i], updated);
237:         }
238: 
239:         return updated;
240:       });
241: =======
     ^
242:    * Manage errors to be thrown
243:    *
244:    * @param {...args} ...args
245:    * @throws {KuzzleError}
246:    */
247: 
248:   throw(...args) {
249: 
250:     const [domain, subdomain, errorName] = args;
251:     if (!errorCodes[domain] [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AParsingError)